smtppipeline.go

```diff
@@ -257,7 +257,7 @@ func (step deliverStep) Pass(ctx *module.DeliveryContext, msg io.Reader) (io.Rea
 		}
 	}
 	received += fmt.Sprintf("\r\n\tby %s (envelope-sender <%s>)", sanitizeString(ctx.OurHostname), sanitizeString(ctx.From))
-	received += fmt.Sprintf("\r\n\twith %s", ctx.SrcProto)
+	received += fmt.Sprintf("\r\n\twith %s id %s", ctx.SrcProto, ctx.DeliveryID)
```

Comment:
[Per RFC 5321](https://tools.ietf.org/html/rfc5321#page-59) "id" should be uppercase

Reply:
```
   ID             = CFWS "ID" FWS ( Atom / msg-id )
                  ; msg-id is defined in RFC 5322 [4]
```
Egh, isn't `Atom` just arbitrary string? (or if you are referring to "id" word before it - in ABNF quoted strings are case-insensitive ).

Reply:
>ABNF strings are case-insensitive

Oh, right -- nevermind then